cheatsheet_tidyverse.Rmd

```diff
@@ -0,0 +1,13 @@
+# Cheatsheet Tidyverse
+
+Jiazhen Li
+
+
+
+
+## 
+
+This is cheatsheet for helping students who start to learn analysis of graph with R. In the class, we learn the tidyverse, a powerful collection of R packages. In this cheatsheet, I pay more attention on two packages: ggplot2 and dplyr. I specifically list functions of dylyr we frequently use and the grammer of graphics in ggplot2 packages. This will be useful for new student to learn.
+
+
+Cheatsheet Tidyverse: https://github.com/Jiazhen980326/contribution-/blob/master/cheatsheet-Jiazhen%20Li%20.pdf
```

Comment:
Links to internal files must contain resources/<project_name>/ in the path, such as: ![Test Photo](resources/sample_project/election.jpg)

Reply:
@Jiazhen980326 Hello, just a reminder to please correct this issue.

Reply:
@Jiazhen980326 Hello, just checking in on this again

Reply:
I am so sorry that I just see the comment. For the first time,  I uploaded the assignment in this section: fall2. Then I found I uploaded to the wrong place because I am in fall 1 section. So I uploaded my contribution again to fall 1 and my contribution has been merged in fall 1. So do I need to correct this issue? Thank you so much.